src/components/textarea/__tests__/__snapshots__/textarea.test.tsx.snap

```diff
@@ -1,5 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
```

Comment:
**note:** New snapshot test to assert that we render two text area's whenever the CSS field-sizing property is not supported.